src/platform/android/DnssdImpl.cpp

```diff
@@ -209,7 +210,20 @@ CHIP_ERROR ChipDnssdBrowse(const char * type, DnssdServiceProtocol protocol, Ine
 
 CHIP_ERROR ChipDnssdStopBrowse(intptr_t browseIdentifier)
 {
-    return CHIP_ERROR_NOT_IMPLEMENTED;
+    VerifyOrReturnError(sMdnsCallbackObject.HasValidObjectRef(), CHIP_ERROR_INCORRECT_STATE);
+
+    JNIEnv * env = JniReferences::GetInstance().GetEnvForCurrentThread();
+
+    env->CallVoidMethod(sBrowserObject.ObjectRef(), sStopBrowseMethod, sMdnsCallbackObject.ObjectRef());
```

Comment:
This is not doing anything with browseIdentifier, right?

So if the core code starts an operational browse, then a commissionable browse, then stops the commissionable browse, what will the behavior here be?

Reply:
Right. In the latest commit, we're now utilizing browseIdentifier to stop DNS-SD browse, similar to Darwin.
For the second question.  Multiple browse operations are currently not possible on Android. We plan to implement that in the future.